packages/preview/fauxreilly/0.1.1/typst.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "fauxreilly"
+version = "0.1.1"
+entrypoint = "lib.typ"
+authors = ["Dei Layborer"]
+license = "GPL-3.0"
+compiler = "0.11.1"
+description = "A package for creating O'Rly- / O'Reilly-type cover pages"
+repository = "https://github.com/dei-layborer/fauxreilly"
+categories = ["components", "layout", "fun"]
+exclude = [
+	"$$$-gimmie.svg",
+	"made-with-(2s)-2,6-diamino-n-[(2s)-1-phenylpropan-2-yl]hexanamide-n-[(2s)-1-phenyl-2-propanyl]-l-lysinamide.svg",
```

You should update the file name here.

```suggestion
	"made-with.svg",
```